packages/nextjs/app/page.tsx

```diff
@@ -218,35 +218,46 @@ const Home: NextPage = () => {
       </div>
 
       <div
-        className={`flex flex-col items-center py-8 gap-12 md:gap-20 bg-white pt-20 text-[#000] ${SpaceGrotesk.className}`}
+        className={`flex flex-col items-center py-10 gap-10 md:gap-16 bg-white pt-20 text-black ${SpaceGrotesk.className}`}
       >
-        <div className="flex flex-col justify-center items-center gap-8 mb-14 px-4">
-          <h1 className="text-center text-2xl lg:text-3xl max-w-md lg:max-w-2xl px-3 m-0">Community</h1>
-          <div className="border border-solid border-[#E2E8F0] rounded-[135px] flex py-8">
-            <div className="border-r-[#E2E8F0] px-20 border-r-[1px]">
-              <Image src="/newsletter.svg" alt="icon" width={130} height={130}></Image>
+        <div className="flex flex-col justify-center items-center gap-6 md:gap-8 mb-10 md:mb-14 px-4">
+          <h1 className="text-center text-2xl md:text-3xl lg:text-4xl max-w-md md:max-w-lg lg:max-w-2xl px-4 m-0">
+            Community
+          </h1>
+
+          {/* Main Community Content */}
```

Comment:
Remove the comment the section {Main Community, Newsletter Icon Section, Community Information Section, Links Section}

All rest is very good, thank you

Reply:
Alright On it now